lang/bg.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Play": "Възпроизвеждане",
+  "Pause": "Пауза",
+  "Current Time": "Текущо време",
+  "Duration Time": "Продължителност",
+  "Remaining Time": "Оставащо време",
+  "Stream Type": "Тип на излъчването",
+  "LIVE": "НА ЖИВО",
+  "Loaded": "Заредено",
+  "Progress": "Прогрес",
+  "Fullscreen": "Голям екран",
+  "Non-Fullscreen": "Спиране на голям екран",
+  "Mute": "Без звук",
+  "Unmuted": "Със звук",
+  "Playback Rate": "Скорост на възпроизвеждане",
+  "Subtitles": "Субтитри",
+  "subtitles off": "Спри субтитрите",
+  "Captions": "Надписи за глухи",
+  "captions off": "Спри надписите за глухи",
+  "Chapters": "Глави",
+  "You aborted the video playback": "Спряхте възпроизвеждането на видеото",
+  "A network error caused the video download to fail part-way.": "Грешка във мрежата провали свалянето на клипа.",
+  "The video could not be loaded, either because the server or network failed or because the format is not supported.": "Видеото не може да бъде заредено, заради проблем със сървъра или мрежата или защото този формат не е поддържан.",
+  "The video playback was aborted due to a corruption problem or because the video used features your browser did not support.": "Възпроизвеждането на видеото беше прекъснато заради проблем със файла или защото видеото използва опции който браузера ви не поддържа.",
```

Comment:
Again, I think 'прекратено' carries a meaning closer to 'aborted' than 'прекъснато'.

'С' should be used instead of 'със' because the next word does not start with a 'с' or a 'з'.

'Ко**и**то' instead of 'който' because 'опции' is plural. And there should be a comma before 'които'.

'Брауз**ъ**р**ът**' instead of 'брауз**е**р**а**' because it's closer to the English pronunciation and the definite article is necessary because the word is a subject.

'ви' should be capitalized if polite form is to be used.